doc/manual/source/release-notes-determinate/changes.md

```diff
@@ -9,3 +9,9 @@ This section lists the differences between upstream Nix 2.24 and Determinate Nix
 * Determinate Nix has a setting [`json-log-path`](@docroot@/command-ref/conf-file.md#conf-json-log-path) to send a copy of all Nix log messages (in JSON format) to a file or Unix domain socket.
 
 * Determinate Nix has made `nix profile install` an alias to `nix profile add`, a more symmetrical antonym of `nix profile remove`.
+
+* `nix-channel` is deprecated, see: https://github.com/DeterminateSystems/nix-src/issues/34
```

Comment:
`channel:` syntax is also deprecated, right?

Reply:
```suggestion
* `nix-channel` and `channel:xxx` url syntax is deprecated, see: https://github.com/DeterminateSystems/nix-src/issues/34
```